assets/js/modules/subscribe-with-google/datastore/constants.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * `modules/subscribe-with-google` data store constants.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const STORE_NAME = 'modules/subscribe-with-google';
+export { STORE_NAME as MODULES_SUBSCRIBE_WITH_GOOGLE };
+
+// Form ID for the module setup form.
+export const FORM_SETUP = 'subscribeWithGoogleSetup';
```

Comment:
This can be removed, since it should no longer be relevant (see my other comment).